app/views/layouts/mailer.text.erb

```diff
@@ -1 +1,29 @@
+<% yield_default :greeting do %>
+  <%# If there is only ONE recipient, it will attempt to extract the name of the recipient %>
+  <% if message&.to&.one? && (name = (@recipient || @user)&.first_name || User.find_by_email_address(message&.to&.first)&.first_name) %>
```

Comment:
Splitting this up between a few lines makes this a bit clearer and negates the need for a comment

Reply:
I ended up dropping it into a helper method

[`5951da7` (#76)](https://github.com/hackclub/hackathons-backend/pull/76/commits/5951da72acbf8f4c91aa97316a43fa9958802008)